src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: 800;
+}
+
+body {
+  font-family: Avenir, sans-serif;
```

Comment:
add fallback font

Reply:
> add fallback font

you meen fallname of font? like this? 

body {
  font-family: **Avenir-Book**, sans-serif;
  font-weight: 300;
  font-size: 16px;
  line-height: 22px;
  color: #3d4e61;
}

Reply:
> add fallback font

where should I add it?

Reply:
I have added the second font to focus:

.searchBar:focus {
  font-family: Avenir, sans-serif;
  font-weight: 800;
  outline: none;
  color: #3d4e61;
  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
  border: 1px solid #e1e7ed;
  border-radius: 3px;
}